### PR TITLE
Pulsar-wide HTML W3C Validation Fixes 

### PR DESCRIPTION
**Pull request by nvasileiadis**
There are a few minor structural fixes to improve the overall code quality and have as less as possible W3C Validation errors in Pulsar.

### CODE REVIEW

File: views/lexicon/tabs/modals.html.twig [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 <p><a data-toggle="modal" href="#myModal" class="btn">Launch Standard Modal</a></p>
 
-<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (Stanton) [inline views/lexicon/tabs/modals.html.twig]: Did `aria-labelledby` throw an info/warning?

File: stylesheets/_component.lexicon.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.example {
+    .wrapper {
```
Comment (Stanton) [inline stylesheets/_component.lexicon.scss]: Could you add some comments here to detail which example UI the styles are for?

File: stylesheets/pulsar.scss [hunk 3]
```diff
@@ -111,6 +111,7 @@ $fa-css-prefix: 'icon';
 @import 'component.icons';
 @import 'component.input-groups';
 @import 'component.labels';
+@import 'component.lexicon';
```
Comment (Stanton) [inline stylesheets/pulsar.scss]: Suspect this would be better at the end of the imports, with the other override/scratch styles, we might need to use the source order to our advantage

File: views/lexicon/forms/checkbox.html.twig [hunk 4]
```diff
@@ -67,7 +67,6 @@
         form.checkbox_inline({
             'id': 'checkbox-7',
             'label': 'Checkbox inline plus extras',
-            'help': 'Help text to give more information about expected input',
```
Comment (Stanton) [inline views/lexicon/forms/checkbox.html.twig]: Why was this removed?

File: views/lexicon/forms/checkbox.html.twig [hunk 5]
```diff
@@ -86,7 +85,6 @@
             'id': 'checkbox-9',
             'label': 'Checkbox inline, indented plus extras',
             'class': 'form__group--indent form__group--full',
-            'help': 'Help text to give more information about expected input',
```
Comment (Stanton) [inline views/lexicon/forms/checkbox.html.twig]: Ditto...

File: views/lexicon/forms/radio.html.twig [hunk 6]
```diff
@@ -66,7 +66,6 @@
         form.radio_inline({
             'id': 'radio-7',
             'label': 'Radio inline plus extras',
-            'help': 'Help text to give more information about expected input',
```
Comment (Stanton) [inline views/lexicon/forms/radio.html.twig]: Why removed?

File: views/lexicon/forms/radio.html.twig [hunk 7]
```diff
@@ -85,7 +84,6 @@
             'id': 'radio-9',
             'label': 'Radio inline, indented plus extras',
             'class': 'form__group--indent form__group--full',
-            'help': 'Help text to give more information about expected input',
```
Comment (Stanton) [inline views/lexicon/forms/radio.html.twig]: Why removed?

File: views/lexicon/tabs/modals.html.twig (hunk 1 above)
Comment (nvasileiadis) [reply views/lexicon/tabs/modals.html.twig]: There is no element with id="myModalLabel". Let me know if you wish to bind this to a different element. (Ref: https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html Example 2)

File: stylesheets/_component.lexicon.scss (hunk 2 above)
Comment (nvasileiadis) [reply stylesheets/_component.lexicon.scss]: Will rework that in ticket #641

File: stylesheets/pulsar.scss (hunk 3 above)
Comment (nvasileiadis) [reply stylesheets/pulsar.scss]: Will rework that in ticket #641

File: views/lexicon/tabs/modals.html.twig (hunk 1 above)
Comment (Stanton) [reply views/lexicon/tabs/modals.html.twig]: It's probably the modal trigger which needs the ID?

File: views/lexicon/forms/checkbox.html.twig (hunk 4 above)
Comment (nvasileiadis) [reply views/lexicon/forms/checkbox.html.twig]: help-w3cerror

Comment (nvasileiadis) [reply views/lexicon/forms/checkbox.html.twig]: This is the state of the element with the 'help' attribute:
help2-w3cerror

Comment (nvasileiadis) [reply views/lexicon/forms/checkbox.html.twig]: This is the state of the element without the 'help' attribute:


No visual regression

File: stylesheets/_component.lexicon.scss (hunk 2 above)
Comment (jamesjacobs) [reply stylesheets/_component.lexicon.scss]: Would prefer it if we scoped class names that were specifically for the lexicon, `.lex-something` for example.

File: views/lexicon/forms/toggle-switch.html.twig [hunk 8]
```diff
@@ -61,8 +61,8 @@
         form.toggle_switch({
             'id': 'toggle-6',
             'label': 'Toggle switch with a longer than expected label which will probably wrap multple lines and push the following input onto the next baseline',
-            'help': 'Example block-level help text here',
-            'guidance': 'Help text to give more information about expected input'
+            'guidance': 'Help text to give more information about expected input',
+            'help': 'Example block-level help text here'
```
Comment (jamesjacobs) [inline views/lexicon/forms/toggle-switch.html.twig]: Curious as to why you needed to change the order here?

File: stylesheets/_component.lexicon.scss (hunk 2 above)
Comment (jamesjacobs) [reply stylesheets/_component.lexicon.scss]: Also, is there any need to nest the selector here, `.example .wrapper` seems cleaner. Surprised the linter didn't flag this, though that might not be enabled yet.

File: views/lexicon/forms/toggle-switch.html.twig (hunk 8 above)
Comment (nvasileiadis) [reply views/lexicon/forms/toggle-switch.html.twig]: It was a restoration of the code. No real difference

Comment (nvasileiadis) [reply views/lexicon/forms/toggle-switch.html.twig]: As for the styles, I have a separate ticket (#641) to work those details separately

File: views/lexicon/tabs/modals.html.twig (hunk 1 above)
Comment (nvasileiadis) [reply views/lexicon/tabs/modals.html.twig]: Just checked it doesn't affect any function of the modal
